tests/vmi_monitoring_test.go

```diff
@@ -78,3 +75,16 @@ var _ = Describe("[sig-compute]Health Monitoring", func() {
 		})
 	})
 })
+
+func withWatchdog() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Devices.Watchdog = &v1.Watchdog{
+			Name: "mywatchdog",
+			WatchdogDevice: v1.WatchdogDevice{
+				I6300ESB: &v1.I6300ESBWatchdog{
+					Action: v1.WatchdogActionPoweroff,
```

Comment:
would we want to parametrize the action ?... Does that make any sense ?

Reply:
The nice thing about defining `withWatchdog()` here, locally, is that I don't need to think much about creating the perfect API. At this time, we have only one user of this function, and what it wants the VM to power off on a watchdog event. I would complicate the function signature only if we ever have a second user for it.